dpm/client/__init__.py

```diff
@@ -49,9 +77,67 @@ def validate(self):
         # should we really check this here? Good question i think ...
         for idx, resource in enumerate(self.datapackage.resources):
             if not exists(resource.local_data_path):
-                raise DpmException('Resource at index %s with name % and path %s does not exist on disk' % (
+                raise DpmException('Resource at index %s with name %s and path %s does not exist on disk' % (
                     idx, resource.name, resource.local_data_path)
                     )
 
         return True
 
+    def ensure_auth(self):
```

Comment:
In a perfect world this should be a "private" method so `_ensure_auth`. However, i don't think this matters for now so let's leave as is.